atmo/settings.py

```diff
@@ -90,6 +90,7 @@
     # setup bootstrap action depends on it to autotune the cluster.
     'MASTER_INSTANCE_TYPE': 'c3.4xlarge',
     'WORKER_INSTANCE_TYPE': 'c3.4xlarge',
+    'USE_SPOT_INSTANCES': config('USE_SPOT_INSTANCES', default='true'),
```

Comment:
Please use `config('USE_SPOT_INSTANCES', default=True, cast=bool)` here since that'll turn the value into a boolean right away